### PR TITLE
Adding Counterspell Saint Augustine

### PR DESCRIPTION
**Pull request by i-suck-at-most-stuff**
Adding Counterspell Saint Augustine page, I followed the templete and im pretty sure i did everything right, i juts have to learn how to make the signup API.

### CODE REVIEW

File: components/SaintAugustine/About.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <div className="space-y-12 max-w-8xl">
+        <h2 className="pb-6 text-4xl text-center uppercase retro text-whitepurp">
+          Make games, <span className="text-pink">meet cool people</span>
+        </h2>
+        <div className="grid grid-cols-1 xl:grid-cols-2 grid-rows-2 gap-12 leading-[3rem] tracking-wider mx-6 2xl:mx-0">
+          <div className="row-span-2 p-8 space-y-12 border-4 border-dashed border-pink md:p-12">
+            <p>
+              {/* TODO: Change `Example City` to the name of your city */}
+              Counterspell Saint Augustine is a beginner friendly game jam for high
+              schoolers happening in Example City where{" "}
```
Comment (samdev-7) [inline components/SaintAugustine/About.js:13]: You forgot to change out "Example City" here.

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/saint-augustine**](https://counterspell-o46wk0lj3.hackclub.dev/saint-augustine) |
|:-:|
|<a href="https://counterspell-o46wk0lj3.hackclub.dev/saint-augustine">Screenshot of /saint-augustine</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/RDjiVHLQGTfmQJof3XqqBDrlfA-qBcTaTay6xwNNnxHqc1P80bC6eKWceP_-GfO5AyyGuNTSG1VHRFLL4A4pLA.png">(view full size)</a> |







Commit 0a5664e23d118544bc271187783c3bcef678c66c (https://counterspell-o46wk0lj3.hackclub.dev).

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Almost there! One quick change.

**Comment by i-suck-at-most-stuff** [comment]
Sorry about that

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**